Use person case v3

emord opened the pull request:
@calellowitz was pretty easy. its a drop in replacement other than agg_awc (seeking_services)

emord committed:
Use personcase_v3 instead of v2

fix personcasev3

emord committed:
Move aggreagtion to person case v3

calellowitz reviewed (approved):
That was scarily simple

calellowitz commented:
I didnt really know how to review the test fixture changes, but the rest seems fine

emord commented:
I used https://github.com/dimagi/commcare-hq/blob/27cb88d9d5ea64e68b84d68ac5e7ca05a3f35bb8/custom/icds_reports/management/commands/migrate_person_cases_ucr.py which is the same command i used on prod and has been backing the test reports, so I'm pretty confident in it

emord committed:
Change != to IS DISTINCT FROM in agg_awc aggregation

After reviewing person_cases_v3 data source, I realized that the only
values will be NULL or 0/1 for registered_status or migration_status.
This is because the app only stores whether a person is not registered
or migrated and a NULL value if its unknown. I have changed our fixture
data source to respect that convention and used IS DISTINCT FROM and !=
to better adhere to the convention that person_cases_v2 data source had
set in place

emord committed:
Fix column names
